src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseHeaderReader.cs

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Justin Van Patten. All rights reserved.
```

Comment:
This is the first time I've seen extra Copyright headers added to the source files.  

@stephentoub Is that allowed per our licensing guidelines for CoreFx?